clients/admin-ui/src/features/common/table/cells.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Tag, Text } from "@fidesui/react";
+import { CellProps } from "react-table";
+
+import { FIELD_TYPE_MAP, MECHANISM_MAP } from "./constants";
+
+export const TitleCell = <T extends object>({
+  value,
+}: CellProps<T, string>) => (
+  <Text fontWeight="semibold" color="gray.600">
+    {value}
+  </Text>
+);
+
+export const WrappedCell = <T extends object>({
+  value,
+}: CellProps<T, string>) => <Text whiteSpace="normal">{value}</Text>;
+
+export const DateCell = <T extends object>({ value }: CellProps<T, string>) =>
+  new Date(value).toDateString();
+
+export const MechanismCell = <T extends object>({
```

Comment:
It is possible to pass in the maps as props at render time.


```typescript
 {
    // @ts-ignore
    cell.column.onToggle
      ? cell.render("Cell", {
          // @ts-ignore
          onToggle: cell.column.onToggle,
        })
      : cell.render("Cell")
  }
```



It might be worth using the pattern you described below with the `EnableCustomFieldCell` component and having each component that uses the Map cell wrap it for itself